identity-service/src/txRelay.js

```diff
@@ -68,6 +68,7 @@ const sendTransactionInternal = async (req, web3, resetNonce = false, txProps, r
     senderAddress,
     gasLimit
   } = txProps
+  const redis = req.app.get('redis')
```

Comment:
could we add a comment here about the decision to use redis here? :)

Reply:
@vicky-g I can if you think it’ll help but this is a pretty common pattern in our code are to use Redis to store metrics/debug info. What would you like the comment to say?

Reply:
hmmm looking more at the redis functionality, i think its clear enough about what the intention here is to not include a comment explanation. i thought itd be a good idea to mention the reason we're using redis here vs prior to the pr where we weren't using it